### ISSUE TITLE
Cannot fread files 3+ Gb on Windows 

### ISSUE THREAD

**Issue by MichaelMoroz1993** [h2oai/datatable#2725]
Hello!

I've faced the following issue:
The fread() function used to read 3+ Gb files returns the error message. At the same moment pandas reads those files without any issues.

[Link ](https://drive.google.com/drive/folders/1ijk-cqAleuXfzIkmNGyDNvPFmZ8b3u32?usp=sharing)to file.

The code and error message are below:

**Code:**
`import datatable as dt` 
`data = dt.fread(file='LF.csv')`

**Error:**
`Traceback (most recent call last):`
 `D:/.../datatable test.py:8 in <module>  data = dt.fread(file='LF.csv')`
`IOError: Unable to obtain size of LF.csv: [errno 132] value too large`

**Environment**
Datatable version - 0.11.0 
Python version - 3.7.2544.0
OS - Windows 10 Pro

Thank you!

**Comment by samukweku** [comment]
I was able to read the data : 

```python
from datatable import fread
df = fread("LF.zip")
df.head()

C0	Row	Date	Timestamp	Sum	ID	Issue	Level	Support	Extras	Left	Skipped	Attempt	Active_support	Platform
▪▪▪▪	▪▪▪▪	▪▪▪▪	▪▪▪▪▪▪▪▪	▪▪▪▪▪▪▪▪	▪▪▪▪	▪▪▪▪	▪▪▪▪▪▪▪▪	▪▪▪▪▪▪▪▪	▪▪▪▪▪▪▪▪	▪▪▪▪▪▪▪▪	▪▪▪▪▪▪▪▪	▪▪▪▪▪▪▪▪	▪▪▪▪	▪▪▪▪
0	0	0	2020-08-01	1596151327000	0	1100357	Fail	27	0	NA	2	0	4		iOS
1	1	1	2020-08-01	1596234522000	0	1101065	Fail	28	0	NA	1	0	2		iOS
2	2	2	2020-08-01	1595571719000	0	1075926	Fail	142	0	NA	1	0	1		iOS
3	3	3	2020-08-01	1582080163000	0	545616	Fail	457	0	NA	1	0	1		iOS
4	4	4	2020-08-01	1594868430000	0	1052669	Fail	262	1	NA	1	1	1		iOS
5	5	5	2020-08-01	1595446708000	0	1070398	Fail	338	0	NA	1	0	3		iOS
6	6	6	2020-08-01	1592420499000	65.72	955438	Fail	445	1	NA	1	1	2		iOS
7	7	7	2020-08-01	1591053071000	0	922235	Fail	618	0	NA	2	0	4		iOS
8	8	8	2020-08-01	1591588299000	2.08391	929291	Fail	457	0	NA	1	1	1		iOS
9	9	9	2020-08-01	1588252416000	0	848026	Fail	807	0	5	1	0	3		iOS

```

Environment: 
datatable - '1.0.0a0+build.1604311178.sam'
Python version - 3.9.0
OS  - Linux

**Comment by pradkrish** [comment]
I am also able to read the data and my env is the same as @samukweku

**Comment by oleksiyskononenko** [comment]
@MichaelMoroz1993 This seems to be the same problem as was reported by @duybluemind1988 on [discussions](https://github.com/h2oai/datatable/discussions/2721). We're currently investigating the reason, but for the moment it seems to be a Windows-specific issue.

**Comment by MichaelMoroz1993** [comment]
@oleksiyskononenko Thank you for your feedback, will track that issue.

**Comment by oleksiyskononenko** [comment]
Closing this issue. The fix will be available in `0.11.1` release we're preparing now.
